packages/wonder-blocks-dropdown/components/check.js

```diff
@@ -38,8 +38,8 @@ export default function Check(props: CheckProps) {
                 disabled
                     ? offBlack32
                     : pressed || hovered || focused
-                        ? white
-                        : offBlack
+                    ? white
```

Comment:
A very strange change from Prettier - I guess this is what they're doing now?

Reply:
Yes, and apparently we are not the only ones that think this is a bit strange: https://github.com/prettier/prettier/issues/5814